src/Gui/Action.cpp

```diff
@@ -247,7 +247,7 @@ void ActionGroup::addTo(QWidget *w)
     // and adding this action to the widget doesn't work.
     if (_dropDown) {
         if (w->inherits("QMenu")) {
-            QMenu *menu = new QMenu(w);
+            auto *menu = new QMenu(w);

```

Comment:
I don't see any really compelling reason to keep the "*" here -- there's no conceivable way `new` is going to return a non-pointer type. I don't feel really strongly about this, but my overall feeling is that with auto we should use the simplest version of it possible, unless the addition provides some actual benefit (e.g. `const` or `&`).